app/controllers/flood_risk_engine/enrollments/addresses_controller.rb

```diff
@@ -0,0 +1,107 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  module Enrollments
+    class AddressesController < ApplicationController
```

Comment:
I took the manual address functionality out of the steps because this "step" is not part of the normal workflow. It seemed to be the easiest way of handling this functionality without having to do major surgery to the steps system. It should also be relatively straight forward to re-use the manual address system for the other address pages.